internal/config/devel_config.go

```diff
@@ -0,0 +1,26 @@
+//go:build !test && development
+// +build !test,development
```

Comment:
Out of curiosity, is this some new/old syntax?

Reply:
yeah, new/old syntax ... my formatter wants to include both :shrug: